web/packages/test/config/beefy-relay.json

```diff
@@ -3,9 +3,8 @@
     "polkadot": {
       "endpoint": "ws://127.0.0.1:9944"
     },
-    "beefy-activation-block": 0,
-    "fast-forward-depth": 20,
-    "update-period": 0
+    "fast-forward-depth": 3000,
+    "update-period": 3000
```

Comment:
These are the config for production to only syncing mandatory/handover commitments.

The downside is for local setup(Rococo) it will also skip the non-handover commitment.  IMO it should be fine because the session length for Rococo is 10 blocks which is not long.

Reply:
Cool, though we store prod config in this repo: https://github.com/Snowfork/snowbridge-infra-suite/

I've added a PR to move the prod config there:
https://github.com/Snowfork/snowbridge-infra-suite/pull/6

So you can revert these changes to the local testnet config.

Reply:
https://github.com/Snowfork/snowbridge/pull/1204/commits/1df8be1578dc3a189199d55d5db953870d636450